recipes/subread/unsignedchar.patch

```diff
@@ -0,0 +1,36 @@
+diff -urNp a/src/fullscan.c b/src/fullscan.c
+--- a/src/fullscan.c	2023-05-07 08:53:37
++++ b/src/fullscan.c	2024-03-21 23:29:44
+@@ -157,7 +157,7 @@ int main (int argc , char ** argv)
+ {
+ 	char index_name [MAX_FILE_NAME_LENGTH];
+ 	char read_str [1208];
+-	char c;
++	int c;
```

Comment:
You could also use `signed char c`

Reply:
The variable is being used exclusively to hold the return value of `getopt()`, which returns `int`. In this case the values returned probably all fit in a `signed char`, but it would be silly to use any type other than `getopt()`'s return type.